gitapp/lib/users_list/users_list.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:gitapp/users_list/user.dart';
+import "package:http/http.dart" as http;
+import 'dart:convert' as convert;
+
+class UsersList {
+  List<User> userlist = List.empty(growable: true);
+  List<User> searchedlist = List.empty(growable: true);
+
+  UsersList() {
+    List<String> usersnames = ['Tytus', 'Romek', 'Atomek', 'Bolek', 'Lolek'];
+    userlist = usersnames
+        .map((name) => User(
+            username: name,
+            avatarUrl:
+                'https://i.kym-cdn.com/entries/icons/original/000/035/310/Peepo_Animation_Banner.jpg'))
```

Comment:
I think that in github API we should have the avatar URL

Reply:
Solved